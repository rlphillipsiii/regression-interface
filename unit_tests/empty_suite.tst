#!/bin/sh

cd $REGRESSION_INT

name="empty suite"
flags=""

source unit_tests/common/setup
source unit_tests/common/dotest
