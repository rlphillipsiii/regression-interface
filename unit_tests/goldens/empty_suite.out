Error: no testcases were found.
